docs/modules/ROOT/pages/erc721.adoc

```diff
@@ -304,48 +216,52 @@ TIP: For information on how to calculate interface IDs, see {computing-interface
 
 ==== Creating a token receiver contract
 
+The Contracts for Cairo `IERC721ReceiverImpl` already returns the correct interface ID for safe token transfers.
+To integrate the `IERC721Receiver` interface into a contract, simply include the ABI embed directive to the implementation and add the `initializer` in the contract's constructor .
```

Comment:
crazy idea, would it make sense to offer the `IERC721Receiver` impl as part of the `ERC721Component` instead of a separate one?

Reply:
bad idea